### PR TITLE
Fix build-and-pack action

### PR DESCRIPTION
**Pull request by ca7alindev**

### CODE REVIEW

File: .github/workflows/releases.yaml
Comment (web7evgheni) [inline .github/workflows/releases.yaml:1]: @ca7alindev Why we need separate builds, next time when we will add new application we will add another build.txt or what, i think we made already GH action where we made all apps to build if they was changed. 
Required just to fix it, but not to rework.

**Review by web7evgheni — CHANGES_REQUESTED** [review]

File: .github/workflows/releases.yaml
Comment (web7evgheni) [reply .github/workflows/releases.yaml:1]: @ca7alindev If structure of backstage is not compatible with build action, we could simple change structure, and do not touch build action at all.

Comment (ca7alindev) [reply .github/workflows/releases.yaml:1]: @web7evgheni that means i need to change location of charts app and backend, also for dockerfiles of them, and that will result in conflicts for Dockerfiles

Comment (web7evgheni) [reply .github/workflows/releases.yaml:1]: @ca7alindev Yes, i think is better, than everytime we add new app, change build script.

Comment (ca7alindev) [reply .github/workflows/releases.yaml:1]: @web7evgheni But i can add additional step that will cover all future structures, and i know how to do it

Comment (web7evgheni) [reply .github/workflows/releases.yaml:1]: @ca7alindev why additional, make it single.

**Pull request closed without merging**